react-ui/src/index.css

```diff
@@ -11,3 +11,13 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+<<<<<<< HEAD
```

Comment:
This seems to be from merge conflict. Resolve it.

Reply:
Done